src/pages/index.js

```diff
@@ -1,50 +1,178 @@
 // ***********************************   DECLARING VARIABLES   ************************************************************
 //IMPORTS
 import './index.css';
-import {initialCards, editButton, profileNameClass, profileDescriptionClass, addPlaceButton, cardModalForm, profileModalForm, cardContainerClass} from '../utils/constants.js';
+import {editButton, profileNameClass, profileDescriptionClass, addPlaceButton, avatarButton, avatarClass, profileModal, profileModalForm, cardModal, cardModalForm, avatarModal, avatarModalForm, cardContainerClass} from '../utils/constants.js';
+import {makePopUpVisible, makeAddPlacePopUpVisible, makeAvatarPopUpVisible, createNewUserInfo, createPopupWithImage, createFormValidator, createPopupWithForm, createPopupWithConfirmation, createNewCard} from '../utils/utils.js';
 import Section from '../components/Section.js';
-import {makePopUpVisible, makeAddPlacePopUpVisible, createNewUserInfo, createFormValidator, createPopupWithForm, createPopupWithImage, createNewCard} from '../utils/utils.js';
+import Api from '../components/Api.js'
 
 
 
 //STORING CLASS INSTANCES
 const imagePopup = createPopupWithImage('#card-modal');
-const userInfo = createNewUserInfo(profileNameClass, profileDescriptionClass);
+const userInfo = createNewUserInfo(profileNameClass, profileDescriptionClass, avatarClass);
 const formValidatorProfileModal = createFormValidator(profileModalForm);
 const formValidatorCardModal = createFormValidator(cardModalForm);
-const addPlacePopup = createPopupWithForm('#add-card-modal', (inputValues) => {
-  const titleValue = inputValues.title;
-  const urlValue = inputValues.url;
-  
-  const cardElement = createNewCard({name: titleValue, link: urlValue});
+const formValidatorAvatarModal = createFormValidator(avatarModalForm);
+
+
+
+
+
+//Class API VARIABLES AND METHODS
+const options = {
+  baseUrl: 'https://around-api.en.tripleten-services.com/v1',
+  headers: {
+    authorization: "4c45d989-e1aa-4bb6-a467-6ac9c46f3dac",
+    "Content-Type": "application/json"
+  }
+}
+
+
+const renderLoading = (isLoading, popup) => {
+  if(isLoading){
+      popup.querySelector('.modal__button').textContent = 'Saving...';
+  } else{
+      popup.querySelector('.modal__button').textContent = 'Save';
+  }   
+}
+
+const loadUserInfoCallback = (data) => {
+  userInfo.setUserInfo(data);
+}
+
+
+const loadCardsCallback = (data) => {
+  const renderCards = new Section({items: data, renderer: (cardItem) => {
```

Comment:
Only one instance of class `Section` should be created in the project because there is only **one** section for the cards.


It will be needed when you add a new card